content/dstu2/authorization.md

```diff
@@ -90,18 +94,12 @@ If successful, the authorization server will return a **200 OK** response with a
 
 The [bearer access token][7] returned from the authorization server is what you provide to the protected resource. If a refresh token was also requested, it will be returned as well. Access tokens are good for 10 minutes and it is recommended refreshing it before use if less than 5 minutes remain before it expires.  
 
-### OpenID Connect ###
-If the scopes "openid" and "profile" were originally provided, an [OpenID Connect][8] id_token will be included per the [SMART][4] specification that includes the user's URL (userfhirurl) as the "profile" claim. This is typically a link to a FHIR<sup>®</sup> standard Practitioner resource.
+## OpenID Connect
```

Comment:
Should we clarify userInfo support (or lack of): http://docs.smarthealthit.org/ (Simple Sign-in section)